backend_new/test/integration/listing.e2e-spec.ts

```diff
@@ -613,6 +613,34 @@ describe('Listing Controller Tests', () => {
     expect(res.body.name).toEqual(val.name);
   });
 
+  it('should successfully process listings that are passed due', async () => {
```

Comment:
I think it might be useful to add a second listing in this test where the applicationDueDate is in the future. That way we can test that only the one listing is updated

Reply:
@YazeedLoonat have you had a chance to look at this comment?

Reply:
sorry I updated this with a new test to cover this case, lemme know what you think!